js/levels/logic/level01.js

```diff
@@ -0,0 +1,13 @@
+import {GameState} from '../../gameState'
+import {LevelPrototype} from './levelPrototype'
+let gState = new GameState().state;
+let _ = require( "lodash" );
+
+let lvl = require( "./../structures/level01.js" );
```

Comment:
now structures are imported from outside, it simplifies the complexity of the level editor. And speretes the logic from content even more